website/docs/essentials/side_effects.mdx

```diff
@@ -153,7 +153,7 @@ would have no mean to know that state has changed.
 The base class extended by your notifier should match that of the provider + modifiers.
 Some examples would be:
 
-- <span style={{ color: colors[0] }}>Notifier</span>Provider -> <span style={{ color: colors[0] }}>Notifier</span>
+- <span style={{ color: colors[0] }}>Notifier</span>Provider -> <span style={{ color: colors[0] }}>Notifier</span>https://camo.githubusercontent.com/
```

Comment:
What is that link?

Reply:
owww sorry it was on my clipboard and got pasted by error (typo on my side i guess)